src/foam/u2/view/ScrollTableView.js

```diff
@@ -364,22 +364,6 @@
         }
       }
     },
-    {
```

Comment:
why do we delete `updateTableHeight`? It's used in couple of places

Reply:
Just checking that, that's why i changed it to do not merge. We shouldn't need to hard code this with the css changes but testing where ever it is used